### ISSUE TITLE
failed to run "03_drawing" example since ellipse API change

### ISSUE THREAD

**Issue by jiyinyiyong** [ggez/ggez#930]
Calling...

https://github.com/ggez/ggez/blob/3723307b6d4f21a14c9e523ae2759f69f133300d/examples/03_drawing.rs#L57-L64

...fails since https://github.com/nical/lyon/issues/606

https://github.com/ggez/ggez/blob/63e246d68bb400509134f217bc565fcd17df31e8/src/graphics/mesh.rs#L169-L182

maybe that example needs to be updated? or the implementation needs updates?

**Comment by PSteinhaus** [comment]
As far as I know this issue was related to a bug in an older version of lyon.

Luckily all of this seems to be fixed in #918 🎊 
I haven't tried the example, but I do assume it just works now. (Of course, if you test it and see otherwise please let me know.)

**Comment by jiyinyiyong** [comment]
I cloned and ran the examples hours ago, the logs:

```
=>> cargo run --example 03_drawing
   Compiling ggez v0.6.0-rc1 (/Users/chen/repo/others/ggez)
    Finished dev [unoptimized + debuginfo] target(s) in 14.65s
     Running `target/debug/examples/03_drawing`
Requested OpenGl 3.2 Core profile, actually got Driver vendor: Intel Inc., renderer Intel(R) HD Graphics 615, version 4.1, INTEL-16.2.16, shading language 4.10.
thread 'main' panicked at 'not implemented', /Users/chen/repo/others/ggez/src/graphics/mesh.rs:182:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
```
=>> git log --oneline
63e246d (HEAD -> master, origin/master, origin/HEAD) Version bump to RC1
cef04ec Merge branch 'devel' of github.com:ggez/ggez into devel
f1df722 Try to add a bit more context to error handling
2ea26a2 Merge pull request #876 from nobbele/glyph-position
9c859ce Merge pull request #891 from erwan-b/add-my-project
```

if I commented out `mb.ellipse(...)` it would run successfully.

> Luckily all of this seems to be fixed in #918 🎊

so the PR not merged yet..

**Comment by jiyinyiyong** [comment]
might be wrong place to report, but as I continue to try examples, I notice `eventloop` example looks strange:

![image](https://user-images.githubusercontent.com/449224/117580039-7aae8d00-b128-11eb-8d00-3118e6ad9003.png)

code after `events_loop.run_return(...)` appears to be unreachable, and if I change that to `.run(...)`(which is actually used in `ggez`) then Rust just tell me the code behind is unreachable.

https://github.com/ggez/ggez/blob/a039db8b94bf56029ab6525a88829fe1e4a259c3/examples/eventloop.rs#L32-L76

probably not ready for me as a beginner... I don't know much about winit for fixing this.

**Comment by jiyinyiyong** [comment]
ggez helped a lot for my experiment project: https://github.com/calcit-lang/calcit-paint . looking forward to fixes and updates.

**Comment by PSteinhaus** [comment]
fixed in #918
